.github/workflows/build.yml

```diff
@@ -1,6 +1,4 @@
 on:
```

Comment:
I can remove this, but there's no reason to build and build+test test on PRs.  We need a separate build step for the release process but can safely remove it from PR runs.